public_html/_routes.php

```diff
@@ -16,7 +16,7 @@
 if (strtolower($path_parts[0]) == 'docs') {
     if (file_exists($docs_md_base . $md_fn)) {
         $markdown_fn = $docs_md_base . $md_fn;
-        $md_github_url = 'https://github.com/nf-core/nf-co.re/tree/master/markdown/' . $md_fn;
+        $md_github_url = 'https://github.com/sanger-tol/pipelines-website/tree/main/markdown/' . $md_fn;
```

Comment:
Could we create a global variable like `$github_base_file_url` which would be something like `https://github.com/sanger-tol/pipelines-website/tree/main` ? so that the GitHub is only defined once